spec/api/houdini/nonprofit_spec.rb

```diff
@@ -77,7 +77,7 @@ def h(h = {})
               password_confirmation: 'doesn\'t match'
           }
       }
-      xhr :post, '/api/v1/nonprofit', input
+      post '/api/v1/nonprofit', params: input, xhr: true
```

Comment:
xhr and the use of positional params in request specs is deprecated so this corrected that.